src/schema/rules/sidecars/derivatives/functional_derivatives.yaml

```diff
@@ -0,0 +1,39 @@
+---
+BandpassFilterOpt:
+  selectors:
+  - dataset.DatasetType == "derivative"
+  - '!(["alff", "falff"].includes(suffix))'
+  fields:
+    BandpassFilter: optional
```

Comment:
Per our discussion today, I think this may need a `level_addendum`. I assume we'll want a more conventional metadata table instead of the current one, which has a "Required for suffix" column instead of the typical "Requirement Level" column.